backstop/tests/tabs.html

```diff
@@ -124,77 +124,53 @@ <h2 id="pill">Pill</h2>
       <li class="iui-tabs-active">
         <a>
-          <!-- Use star.svg from generic-icons package. -->
+          <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
           <svg
-            width="16px"
-            height="16px"
             viewBox="0 0 16 16"
             aria-hidden="true"
           >
-            <path d="M13,1.5c0.8271,0,1.5,0.6729,1.5,1.5v10c0,0.8271-0.6729,1.5-1.5,1.5H3c-0.8271,0-1.5-0.6729-1.5-1.5V3
-         c0-0.8271,0.6729-1.5,1.5-1.5H13 M13,0H3C1.3432,0,0,1.3432,0,3v10c0,1.6568,1.3432,3,3,3h10c1.6568,0,3-1.3432,3-3V3
-         C16,1.3432,14.6568,0,13,0L13,0z M9.5697,6.3099L8.3439,3.7415C8.2702,3.5481,8.0537,3.451,7.8603,3.5246
-         c-0.0999,0.038-0.1789,0.117-0.2169,0.2169L6.4176,6.3099L3.6157,6.7186C3.2654,6.7769,3.2654,7.1272,3.4989,7.3607L5.4836,9.287
-         l-0.5254,2.8019c-0.0584,0.4086,0.2919,0.467,0.5254,0.3502l2.51-1.2842l2.51,1.2842c0.1537,0.1071,0.3651,0.0692,0.4722-0.0845
-         c0.0539-0.0773,0.0731-0.1736,0.0531-0.2657L10.562,9.287l2.0431-1.9847c0.1137-0.1832,0.0573-0.4239-0.1259-0.5376
-         c-0.0333-0.0207-0.0696-0.0362-0.1076-0.0461L9.5697,6.3099z" />
+            <path
+              d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+            />
           </svg>
         </a>
       </li>
       <li>
         <a>
-          <!-- Use star.svg from generic-icons package. -->
+          <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
           <svg
-            width="16px"
-            height="16px"
             viewBox="0 0 16 16"
             aria-hidden="true"
           >
-            <path d="M13,1.5c0.8271,0,1.5,0.6729,1.5,1.5v10c0,0.8271-0.6729,1.5-1.5,1.5H3c-0.8271,0-1.5-0.6729-1.5-1.5V3
-         c0-0.8271,0.6729-1.5,1.5-1.5H13 M13,0H3C1.3432,0,0,1.3432,0,3v10c0,1.6568,1.3432,3,3,3h10c1.6568,0,3-1.3432,3-3V3
-         C16,1.3432,14.6568,0,13,0L13,0z M9.5697,6.3099L8.3439,3.7415C8.2702,3.5481,8.0537,3.451,7.8603,3.5246
-         c-0.0999,0.038-0.1789,0.117-0.2169,0.2169L6.4176,6.3099L3.6157,6.7186C3.2654,6.7769,3.2654,7.1272,3.4989,7.3607L5.4836,9.287
-         l-0.5254,2.8019c-0.0584,0.4086,0.2919,0.467,0.5254,0.3502l2.51-1.2842l2.51,1.2842c0.1537,0.1071,0.3651,0.0692,0.4722-0.0845
-         c0.0539-0.0773,0.0731-0.1736,0.0531-0.2657L10.562,9.287l2.0431-1.9847c0.1137-0.1832,0.0573-0.4239-0.1259-0.5376
-         c-0.0333-0.0207-0.0696-0.0362-0.1076-0.0461L9.5697,6.3099z" />
+            <path
+              d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+            />
           </svg>
         </a>
       </li>
       <li>
         <a>
-          <!-- Use star.svg from generic-icons package. -->
+          <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
```

Comment:
![image](https://user-images.githubusercontent.com/81580355/116877655-e7a4bd00-ac26-11eb-8b06-0ee9abeb2e89.png)

Was these before too?

Reply:
Removed the duplicate `<hr />` within `table.scss`.  I'm not sure why this response is tied to `tabs.html` but I have also encountered a GitHub bug where I respond to something & it gets tied to a completely unrelated file. 🤷🏻‍♂️